deps/llvm.mk

```diff
@@ -94,6 +94,7 @@ LLVM_CMAKE += -DLLVM_TARGETS_TO_BUILD:STRING="$(LLVM_TARGETS)" -DCMAKE_BUILD_TYP
 LLVM_CMAKE += -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD:STRING="$(LLVM_EXPERIMENTAL_TARGETS)"
 LLVM_CMAKE += -DLLVM_ENABLE_LIBXML2=OFF -DLLVM_HOST_TRIPLE="$(or $(XC_HOST),$(BUILD_MACHINE))"
 LLVM_CMAKE += -DLLVM_ENABLE_ZLIB=ON -DZLIB_LIBRARY="$(build_prefix)/lib"
+LLVM_CMAKE += -DLLVM_ENABLE_ZSTD=OFF
```

Comment:
If you have libzstd somewhere on the system that's being searched for libraries, LLVM will try to pull it in, which causes other linking issues. AFAICT we don't need this set anyway so disabling it shouldn't break anything.